2024-09-07 21:58:29.758336;

Error sending result: '<multiprocessing.pool.ExceptionWithTraceback object at 0x7f61827fbd70>'. Reason: 'TypeError("cannot pickle '_thread.RLock' object")'